src/components/board/board.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "../../styles/styles.css";
+import Square from "../square/square";
+
+
+const files = ["a", "b", "c", "d", "e", "f", "g", "h"];
+const ranks = [1, 2, 3, 4, 5, 6, 7, 8];
+
+function makeBoard(ranks, files) {
+  return ranks.reduce((board, rank) => {
+    files.forEach(file => {
+      const id = `${file}${rank}`;
+      board.push({ id: id, piece: null, color: "white" });
+    });
+    return board;
+  }, []);
+}
+
+const board = makeBoard(ranks, files);
+console.log(board);
+
+export default function Board() {
+  return (
+    <div className="wrapper">
+      <div className="top">
+        <div className="wrapper-inner">
+          <div className="box-inner">A</div>
```

Comment:
For these files - you could make the values in the files array to be upper case and loop through that array similar to the board.

`{files.map(value => (
     <div className="box-inner">{value}</div>
 ))}`

Reply:
+1